### ISSUE TITLE
Can't import via SPM

### ISSUE THREAD

**Issue by simonmcl** [airgap-it/beacon-ios-sdk#1]
Can't import via SPM due to an underlying issue with the way sodium is packaged. There is a work around, but the owner refuses to integrate as he wants Apple to fix the underlying issue in Xcode instead. Its listed as an "known issue" in Xcode 13 so will hopefully be fixed with that release.

![Screenshot 2021-08-06 at 09 44 23](https://user-images.githubusercontent.com/8058386/128483577-f2c619f3-6a84-4efc-88e3-a504863cfaf8.png)

Someone else made a fork that fixes the issue in most cases for now. You can update your dependency to use:

```
.package(name: "Sodium", url: "https://github.com/junelife/swift-sodium.git", .branch("spm")),
```

and remove the line 

```
.product(name: "Clibsodium", package: "Sodium"),
```

And take note to revert it back when the issue is addressed.

**Comment by jsamol** [comment]
The workaround has been applied in `v2.0.0`.

**Comment by simonmcl** [comment]
@jsamol FYI, the issue has been resolved in Xcode 13. I've updated all my dependencies to use the official version (including a fork of beacon), and i'm able to submit builds to TestFlight again

**Comment by jsamol** [comment]
@simonmcl thanks for letting me know, we've just released a new version `3.0.0` and I've already removed the workaround 😄
